### PR TITLE
March newsletter

### PR DESCRIPTION
**Pull request by bethaitman**

### CODE REVIEW

File: docs/blog/newsletter-march-2021.rst [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+.. post:: March 05, 2021
+   :tags: newsletter
+
+######################################
+Write the Docs Newsletter – March 2021
+######################################
+
+Hey everyone! It's your friendly neighbourhood WTD newsletter team, with our March edition.
+
+Spring is here and the year is racing on, and can you believe it's getting close to conference time? We've just announced `speakers for our virtual Portland conference </conf/portland/2021/news/announcing-speakers/>`__, which is coming up on 25th-27th April. If you think those speakers look great (which of course you do!), you can get your tickets `here </conf/portland/2021/tickets/>`__.
```
Comment (plaindocs) [inline docs/blog/newsletter-march-2021.rst]: What do you think of this instead?

```suggestion
Spring is here and the year is racing on, and can you believe it's getting close to conference time? We've just announced `speakers for our virtual Portland conference </conf/portland/2021/news/announcing-speakers/>`__, which is coming up on 25th-27th April. If you think those speakers look great (which of course you do!), `get your tickets </conf/portland/2021/tickets/>`__ now.
```

File: docs/blog/newsletter-march-2021.rst [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+And if Portland can't come soon enough, the West Coast Quorum meetup is holding its first virtual event this month! The event is about "The Product is Docs: A Look Inside" and will be held on Thursday 18th March at 19:00 Pacific Time. If you're interested, `sign up to attend <https://www.meetup.com/virtual-write-the-docs-west-coast-quorum/events/276616460/>`__.
```
Comment (plaindocs) [inline docs/blog/newsletter-march-2021.rst]:
```suggestion
And if Portland isn't soon enough for you, the West Coast Quorum meetup is holding its first virtual event this month! The event is about "The Product is Docs: A Look Inside" and will be held on Thursday 18th March at 19:00 Pacific Time. If you're interested, `sign up to attend <https://www.meetup.com/virtual-write-the-docs-west-coast-quorum/events/276616460/>`__.
```

File: docs/blog/newsletter-march-2021.rst [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+Most agreed it's not worth making special effort to edit for only this offense. As some said, "it's in the backlog's backlog's backlog ..." So while nobody seems to like "etc", most acknowledged they let such things go more often than not, unless thez can see that these terms have become real stumbling blocks for users.
+
+---------------------------
+How to indicate UI elements
+---------------------------
+
+What's your style for marking UI elements in text? It's common to want to emphasize that something is (for example) a button so it stands out, whether you write it as the Save, *Save* or **Save** button. Why? It helps users to scan the document; and if your button is well-named, it should also match up to something the user is trying to find for the task at hand. Plus if UI text is in sentence case, this can result in confusing phrasing in docs, and emphasis can help the reader parse the sentence.
```
Comment (plaindocs) [inline docs/blog/newsletter-march-2021.rst:49]: Should Save *Save* and **Save** be in `code`? Works either way.

**Review by plaindocs — APPROVED** [review]
Looks great to me.

File: docs/blog/newsletter-march-2021.rst (hunk 3 above)
Comment (bethaitman) [reply docs/blog/newsletter-march-2021.rst:49]: I think I'll keep it as is.

**Pull request merged**